atheaderservice/Dockerfile

```diff
@@ -15,7 +18,6 @@ ARG LSSTTS_RPM_VERSION=$LSSTTS_SAL_VERSION-$LSSTTS_XML_VERSION.el7
 # Versions
 ARG HEADERSERVICE_VERSION
 ARG SALPYTOOLS_VERSION
-ARG INSTALL_PATH=/opt/lsst
```

Comment:
Was removing this meant to disable the capability of changing it on build so that it remains the same every time one builds?